JamulatorUnityProject/Assets/Scripts/Audio/AudioUtility.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+
+public static class AudioUtility
```

Comment:
might be worth putting the WaitBeforePlayClip(delay, clip) function in here since you've written it out a couple times

Reply:
yeah, good idea